skbeam/core/accumulators/binned_statistic.py

```diff
@@ -620,7 +620,7 @@ def __call__(self, values, statistic=None):
             The values of the selected statistic in each bin.
         """
         # check for what I believe could be a common error
-        if values.shape != self.expected_shape:
+        if tuple(values.shape) != tuple(self.expected_shape):
```

Comment:
I don't understand the change here. `self.expected_shape` is a tuple according to the docstring in the `__init__` and `values.shape` is coming off of the numpy array, so that API is pretty well fixed. Are you finding that you hit an issue `if values.shape != self.expected_shape`?